internal/web_server/web/views/pages/quiz_pages/user_profile.templ

```diff
@@ -0,0 +1,71 @@
+package quiz_pages
+
+import "github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/layout_components"
+import "github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/icons"
+import "github.com/Molnes/Nyhetsjeger/internal/models/users"
+
+//Javascript function to show the delete confirmation modal
+
+script showDeleteConfirmationModal() {
+    document.addEventListener("htmx:confirm", function(e) {
+
+        if (e.target.id === "delete-button") {
+
+        e.preventDefault();
+        document.getElementById("modal").classList.remove("hidden");
+        document.getElementById("modal").classList.add("flex");
+
+        document.getElementById("accept-button").addEventListener("click", function() {
+            document.getElementById("modal").classList.add("hidden");
+            document.getElementById("modal").classList.remove("flex");
+            e.detail.issueRequest(true)
+        });
+
+        document.getElementById("refuse-button").addEventListener("click", function() {
+            document.getElementById("modal").classList.add("hidden");
+            document.getElementById("modal").classList.remove("flex");
+        }); 
+    }
+    })
+}
+
+templ UserProfile(u *users.User) {
+	@layout_components.BaseLayout("Nyhetsjeger - Profil") {
+		@showDeleteConfirmationModal()
+		<div class="h-screen flex justify-center items-center">
+			<form class="w-1/3 h-3/4 bg-purple-400 rounded-lg flex flex-col items-center" action="" method="post">
```

Comment:
No point in having action and post when using HTMX?

Reply:
This is a form. I do believe it requires method either way. Empty actions means current path. I do not know how else you can send form data with HTMX.

Reply:
Everywhere you use `hx-post=""`, `hx-patch=""`, `hx-delete=""` you are already overriding the form action and method. But I suppose there is no harm in letting it stay, just a bit redundant.